docs/INTEGRATION_FOREGROUND_FILTERS.md

```diff
@@ -0,0 +1,123 @@
+## Foreground Filters
+Foreground filters provide a way for pre-applying a filter on page load. The applied filter will be applied to the URL and can be removed as any other applied filter would. Foreground filtering is accomplished by setting the inital UrlManager state; this can be used for setting various states, but only filtering will be covered in this document.
```

Comment:
Consider renaming "covered in this document" to "covered in this example"?

Reply:
All examples in the document only cover filtering - do you think it should still be re-worded?

Reply:
document sounds good